### DISCUSSION TITLE
Newest ONNXruntime with CUDA10.2 and CUDNN 7.5 Support

### DISCUSSION THREAD

**Discussion by endoskop27** [microsoft/onnxruntime#16191] [Other Q&A]
Dear community, 
is there any hint where to find a wheel with a newer version of onnxruntime compiled against cuda10.2 and cudnn 7.5 support?
Im using Nvidia drive Xavier which seems to be outdated in terms of cuda+cuddn. 
thank you!

**Comment by xadupre** [comment]
onxruntime is released with the same CUDA version as pytorch (currently 11.8). You need to compile the code to compile with another version of cuda.

**Reply by snnn** [reply]
Jetson should support CUDA 11.4.  Have you tried the latest Jetson SDK?

**Reply by endoskop27** [reply]
thanks for the replies. Unfortunately I am using the NVIDIA DRIVE and not the JETSON Platform. 
I tried to figure out wether both are different at all and it seems there are some differences.
From documentation the Xavier HW limits the versions of CUDA which does not allow to use newer CUDA versions on this Hardware. 

don't know wether this is technical issue or just another marketing thing from Nvidia.
Does anybody of you have some insights?
best

**Reply by snnn** [reply]
ONNX Runtime 1.6 should work with CUDA 10.2 fine. I'm not sure if any of the newer version still works.
